packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -1289,6 +1291,15 @@ contract Deploy is Deployer {
             customGasTokenAddress = cfg.customGasTokenAddress();
             IERC20 token = IERC20(customGasTokenAddress);
             initialBalance = token.balanceOf(optimismPortalProxy);
+
+            uint256 balanceStorageSlot = 61; // slot of _balance variable
```

Comment:
Is there a way we could make newly introduced incompatibilities detectable? 
As soon as the slot number changes upstream, it's possible we don't notice modifying the wrong slot.

Reply:
I added chain assertion